service/src/main/java/org/cbioportal/genome_nexus/service/cached/BaseCachedExternalResourceFetcher.java

```diff
@@ -24,8 +24,16 @@
     protected ExternalResourceFetcher<T> fetcher;
     protected ResourceTransformer<T> transformer;
 
-    public BaseCachedExternalResourceFetcher(ResourceTransformer<T> transformer)
+    public BaseCachedExternalResourceFetcher(String collection,
+                                             R repository,
+                                             Class<T> type,
+                                             ExternalResourceFetcher<T> fetcher,
+                                             ResourceTransformer<T> transformer)
```

Comment:
Updated the constructor in order to force the child classes to provide these 5 arguments.